BZ2070625: Encrypting EBS instance volumes with a KMS key (#48347)
openshift/openshift-docs: storage/persistent_storage/persistent-storage-aws.adoc

Persistent volumes are not bound to a single project or namespace; they can be
shared across the {product-title} cluster.
Persistent volume claims are specific to a project or namespace and can be

> requested by users.You can define a KMS key to encrypt EBS instance volumes. 

Reviewer: ```suggestion
requested by users.You can define a KMS key to encrypt AWS container persistent volumes. 
```

Author: Addressed

> requested by users. You can define a KMS key to encrypt container persistent volumes on AWS. 

Reviewer 2: ```suggestion
requested by users. You can define a KMS key to encrypt container-persistent volumes on AWS. 
```

Reviewer 2: See ISG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-hyphens#compound-words

Author: Addressed

> requested by users. You can define a KMS key to encrypt container persistent volumes on AWS. 

Reviewer 2: ```suggestion
requested by users. You can define a KMS key to encrypt container-persistent volumes on AWS. 
```

Reviewer 2: See ISG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-hyphens#compound-words

Author: Addressed
